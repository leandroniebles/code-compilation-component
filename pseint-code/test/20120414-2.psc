Proceso A
  Mientras >4 hacer
  FinMientras
  Mientras 4< hacer
  FinMientras
FinProceso
