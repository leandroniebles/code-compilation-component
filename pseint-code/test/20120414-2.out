=== Line 2: SynError 234
=== Line 4: SynError 237
